pkg/cri/server/sandbox_run.go

```diff
@@ -299,8 +341,8 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 			deferCtx, deferCancel := ctrdutil.DeferContext()
 			defer deferCancel()
 			// Cleanup the sandbox container if an error is returned.
-			if _, err := task.Delete(deferCtx, WithNRISandboxDelete(id), containerd.WithProcessKill); err != nil && !errdefs.IsNotFound(err) {
-				log.G(ctx).WithError(err).Errorf("Failed to delete sandbox container %q", id)
+			if _, cleanupErr = task.Delete(deferCtx, WithNRISandboxDelete(id), containerd.WithProcessKill); cleanupErr != nil && !errdefs.IsNotFound(cleanupErr) {
```

Comment:
need to cleanup cleanupErr if it is not found error here~

Reply:
Good catch. Will fix.